recipe/meta.yaml

```diff
@@ -0,0 +1,95 @@
+{% set name = "pymoo" %}
+{% set version = "0.6.1.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 65637ea2a49fb836b638fd5d63a5da4f3bfa713fe8b283e0c22c5e9af3b024db
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - pip
```

Comment:
```suggestion
    - cython
    - numpy {{ numpy }}
    - pip
```
As you can compile this package you need `cython` and `numpy` in `host`, see https://github.com/anyoptimization/pymoo/blob/0.6.0.1/setup.py#L160-L185 and https://github.com/anyoptimization/pymoo/blob/819d68ba0e6107b5ff5ff55d91b94e063f27d834/setup.py#L204-L206
The links are for v0.6.0.1 but all the same, applies for [v0.6.1.1](https://pypi.org/project/pymoo/#files) as the tag isn't available on Github.